README.md

```diff
@@ -229,7 +229,7 @@ GitHub Actions is configured to run these linters on every pull request, so you
 There are a few ways to manage this:
 
 1. Run `yarn lint:write` in the `frontend/` dir, and `./gradlew spotlessApply` in the `backend/` dir, before every commit
-1. Enable the optional pre-commit hook by running `yarn install` in the root dir
+1. Enable the optional pre-commit hook by running `yarn lefthook install` in the root dir
```

Comment:
Oh hmm probably we need to run both. I had run yarn install but the pre-commit hooks didn't start running for me until I specifically ran `yarn lefthook install`.

Reply:
Running `yarn install` is already part of the install steps on L49, so I think this would just be to _enable_ it. I'll add something to make that more clear.